src/Views/BundleView.vala

```diff
@@ -0,0 +1,138 @@
+/*
```

Comment:
Instead of duplicating this whole class with slightly different logic, it would be better to just have two separate methods in the existing class (i.e. `display_ref_details` and `display_bundle_details`). This is especially important since the strings are basically exactly the same, so we don't have to update them in two places and have translators translate them twice.